resources/lib/playitem.py

```diff
@@ -63,7 +64,7 @@ def handle_now_playing_result(self, result):
 
                 # Get current episodeid
                 current_episode_id = self.api.get_episode_id(
-                    showid=str(self.state.tv_show_id), show_season=current_season_id,
+                    showid=from_unicode(self.state.tv_show_id), show_season=current_season_id,
```

Comment:
Again, because you imported `unicode_literals` we assume every string is unicode. We only decode and encode when inputting or outputting data from and to xbmc api on the edges of our program.

Reply:
This should stay `str()` in fact because we want to case an int to a string. I already fixed this.